TTKDTrainingApp/src/utilities/categories.js

```diff
@@ -0,0 +1,5 @@
+const categories = require('../mock_data/categories.json').categories;
+
+export const getCategories = (parentId = null) => {
```

Comment:
would understand if we wanted to call this something different like `getSubCategories`, as technically the categories we're grabbing are subCategories of the parentId passed in